docs/Tutorials/Create-Private-Network.md

```diff
@@ -56,6 +57,7 @@ Copy the following genesis definition to a file called `privateNetworkGenesis.js
 {
   "config": {
     "ethash": {
+      "fixeddifficulty": 1000
```

Comment:
Why 1000 ? why not 100 like in development network dev.json ?

Reply:
As discussed - not quite as fast so you can see transactions going into the blocks.